pages/articles.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+    <div class="pa-5">
+        <div v-if="loading">
+            <Loading :active="loading"
+                :can-cancel="false"
+                :is-full-page="true"
+                color="#007bff"
+            />
+        </div>
+        <div v-else>
+            <v-row align="center" class="mx-0">
+                <v-col :cols="4" v-for="article, index in currentArticleList" :key="index">
+                    <ArticleCard
+                        :key="index"
+                        :articleData="article"
+                    />
+                </v-col>
+            </v-row>
+
+            <v-pagination
+                circle
+                v-model="currentPage"
+                :length="totalPages"
+                @input="onPageNumberClick"
+            ></v-pagination>
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+// Library imports
+import { Vue, Component } from 'vue-property-decorator'
+import Loading from 'vue-loading-overlay'
+import 'vue-loading-overlay/dist/vue-loading.css'
+
+// Components imports
+import ArticleCard from '~/components/article_card.vue'
```

Comment:
Components directory is auto imported by default .

Reply:
Fixed.